local-volume/README.md

```diff
@@ -167,6 +179,9 @@ $ kubectl create -f provisioner/deployment/kubernetes/example-storageclass.yaml
 
 #### Option 1: Using the local volume static provisioner 
 
+**Note:** Running provisioner v2.1.0 or later against a Kubernetes cluster <
```

Comment:
Maybe move this further up

Reply:
nit: change ` cluster <` to `cluster with a version prior to`

Reply:
Re: moving it up further. 

In this area of the test, let's add a note about the version the k8s cluster (v1.10) and provisioner (v2.1.0) we recommend people use and to which this section's text is correct. We can include an example for pre-1.10 in-line or link out to it as a known issue.

Reply:
added a note in the beginning and also in the changelog